_posts/0000-01-09-krutin86.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+I miss Cheeseburgers
```

Comment:
Do you prefer a double cheeseberger?  ... with bacon?

Reply:
YES!  But no tomatoes